dashboard/app/models/peer_review.rb

```diff
@@ -126,15 +126,17 @@ def mark_user_level
     # Need at least `REVIEWS_FOR_CONSENSUS` reviews to accept/reject
     return unless reviews.size >= REVIEWS_FOR_CONSENSUS
 
-    # TODO: Add an else clause with find_or_create PeerReview assigned to the
-    # instructor.
     if reviews.all?(&:accepted?)
       user_level.update!(best_result: Activity::REVIEW_ACCEPTED_RESULT)
       update_column :audit_trail, append_audit_trail("COMPLETED by user id #{reviewer_id}")
     elsif reviews.all?(&:rejected?)
       user_level.update!(best_result: Activity::REVIEW_REJECTED_RESULT)
       update_column :audit_trail, append_audit_trail("REJECTED by user id #{reviewer_id}")
     else
+      # No consensus: escalate the review (i.e. create an escalated review based on this one)
+      escalated_review = dup
+      escalated_review.assign_attributes(status: 'escalated', reviewer: nil)
+      escalated_review.save!
```

Comment:
@mehalshah when this new escalated peer review is reviewed, will this callback again log no consensus and re-escalate? Should the above logic be different? Or are `escalated` reviews handled differently somewhere?

Maybe we need more unit tests here, for the next step, reviewing the escalated review?

Reply:
Line 102 handles this.

While you are here, could you append to the audit trail in the line 102 block?